cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -269,7 +269,7 @@ include_directories(SYSTEM ${PQXX_INCLUDE_DIRECTORIES})
 list(APPEND TERRIER_LINK_LIBS ${PQXX_LIBRARIES})
 
 # LLVM 8.0
-find_package(LLVM 8.0 REQUIRED CONFIG)
```

Comment:
This is an artifact of when we started by supporting LLVM 6+, but once we moved to requiring 8 only we left the logic. I'm concerned that removing that will cause issues for people who have multiple versions of LLVM installed. Let's restrict the scope of this PR just to GCC fixes, and revisit LLVM later.

Reply:
Makes sense. This should be fine for now, then.